xAPI.md

```diff
@@ -600,12 +600,18 @@ the resulting document stored in the LRS is:
 
 ## 2.0 Statements  
 
-### 2.1 Purpose
-
 ###### Description 
 The Statement is the core of the xAPI. All learning events are stored as Statements.
 A Statement is akin to a sentence of the form "I did this".
 
+### 2.1 Purpose
+
+Statements are the evidence for any sort of experience or event which is to be tracked in xAPI. Statements can be expressed 
```

Comment:
I think "expressed" might be too strong; there are already a lot of people confused over if the natural language forms count as actual statements, when they definitely aren't -- they're just approximations. Maybe switch to "approximated" or "described"?